docsets/Powershell/docset.json

```diff
@@ -1,18 +1,21 @@
 {
     "name": "Powershell",
-    "version": "0.6-2-g27870a9/18-06-06",
+    "version": "/22-07-24",
     "archive": "Powershell.tgz",
```

Comment:
You need to include the archive at this path as well. I'll copy the archive from `versions/7.2/Powershell.tgz` this time.

Reply:
So I should change that line to `"version": "7.2",`, right?

Reply:
No, it's not about the version. It's about the archive itself. The archive for the main docset (located at path `docsets/Powershell/Powershell.tgz`) is separate from archives within `specific_versions`.